__tests__/schema/ai.ts

```diff
@@ -51,8 +46,10 @@ const query = new Client({ userId: user.id }).prepareQuery({
 })
 
 beforeAll(() => {
-  mockOpenAIServer((_req, res, ctx) => {
-    return res(ctx.status(200), ctx.json(mockedOpenAiResponse))
+  mockOpenAIServer(() => {
+    return HttpResponse.json(mockedOpenAiResponse, {
+      status: 200,
+    })
```

Comment:
```suggestion
    return HttpResponse.json(mockedOpenAiResponse)
```

Status = 200 should be the default // maybe it is better to make it explicit here...